src/backend/protocol.ts

```diff
@@ -31,11 +31,14 @@ export async function handleProtocol(window: BrowserWindow, args: string[]) {
   }
 
   if (command === 'launch') {
-    const game =
-      getGame(arg, 'legendary').getGameInfo() ||
-      getGame(arg, 'gog').getGameInfo()
+    let game = getGame(arg, 'legendary').getGameInfo()
 
-    if (!game) {
+    // getGameInfo returns {} so we need to check for underlying value
+    if (!game?.app_name) {
+      game = getGame(arg, 'gog').getGameInfo()
+    }
```

Comment:
Maybe something like this would be more elegant:
```ts
const allRunners: Runner[] = ['legendary', 'gog', 'sideload']
const game = allRunners
  .map((runner) => getGame(arg, runner).getGameInfo())
  .filter(({ app_name }) => app_name)
  .shift()

Reply:
totally forgot sideload is a thing, yeah that would be much more scalable

Reply:
However getGame doesn't handle sideload runner

Reply:
Well, nothing says that it *can't*